test/module/libs/common/result_test.cpp

```diff
@@ -24,16 +24,28 @@ using iroha::expected::Value;
 using iroha::expected::makeError;
 using iroha::expected::makeValue;
 
+constexpr auto error_case_message = "Unexpected error case";
+constexpr auto value_case_message = "Unexpected value case";
+constexpr auto first_function_call_message = "first function";
+constexpr auto error_message = "error message";
+
+// Function which returns function which when invoked, fails the test
+template <typename T>
+auto makeFailCase(const std::string &error_message) {
+  return [&error_message](T t) { FAIL() << error_message; };
```

Comment:
Is it formatted with clang? Afair it enforces new lines for non empty blocks inside braces

Reply:
yes it is formatted with clang format